src/functions/schedule-timed-curations/handler.ts

```diff
@@ -18,14 +18,15 @@ const scheduleTimedCuration = async () => {
   const timedCurations = await timedCurationsService.listTimedCurations();
 
   await Promise.all(timedCurations.map(async timedCuration => {
-    const { id, curationId, startTime, endTime, hidden, promoted, queries } = timedCuration;
+    const { id, curationId, startTime, endTime, hidden, promoted, queries, isManuallyCreated } = timedCuration;
 
     const now = new Date();
     const start = parseDate(startTime);
     const end = parseDate(endTime);
     const active = start.getTime() <= now.getTime() && end.getTime() >= now.getTime();
+    const alwaysActive = isManuallyCreated && !startTime && !endTime;
 
-    if (!curationId && active) {
+    if (!curationId && active || !curationId && alwaysActive) {
```

Comment:
This should be simplified. This whole if-else could probably be even simpler, but at least this should be changed.
```suggestion
    if (!curationId && (active || alwaysActive)) {
```